export const throwError = () => {
    throw new Error('FS operation failed')
}